src/Features/Features.php

```diff
@@ -232,14 +232,18 @@ public static function add_features_settings( $settings, $current_section ) {
 	/**
 	 * Conditionally loads the beta features tracking modal.
 	 *
-	 * @param array $settings Settings.
-	 * @return array
+	 * @param string $hook Page hook.
 	 */
-	public static function maybe_load_beta_features_modal( $settings ) {
+	public static function maybe_load_beta_features_modal( $hook ) {
+		if (
+			'woocommerce_page_wc-settings' !== $hook
```

Comment:
This will load on all the settings pages.  Maybe we could check if we're on the right tab and section before enqueuing this script?

`isset( $_GET['tab'] ) && 'advanced' === $_GET['tab'] && isset( $_GET['section'] ) && 'features' === $_GET['section'] `

Reply:
Thanks, will do that.
Not that it's super important atm, but would the above also work for when we use the new settings?

Reply:
@joshuatf Updated in https://github.com/woocommerce/woocommerce-admin/pull/6358/commits/bcd4f337c64625ba2ba8f73493d43d3937776035